src/features/classification/applied-by-ai-classification-reason/AppliedByAiClassificationReason.tsx

```diff
@@ -30,8 +30,18 @@ const AppliedByAiClassificationReason = ({
         <FormattedDate value={modifiedDate} month="long" year="numeric" day="numeric" />
     );
 
+    const hasContent = Boolean(answer) || Boolean(citations);
+
     return (
-        <Card className={classNames('bdl-AppliedByAiClassificationReason', className)}>
+        <Card
+            className={classNames(
+                'bdl-AppliedByAiClassificationReason',
+                {
+                    'bdl-AppliedByAiClassificationReason--noContent': !hasContent,
+                },
```

Comment:
this seems like it might be better as a state rather than modifier:

```suggestion
                'bdl-AppliedByAiClassificationReason',
                {
                    'is-empty': !hasContent,
                },
```

then the SCSS is:
```
.bdl-AppliedByAiClassificationReason {
    &.is-empty {
        ...
    }
}

```

Reply:
Hmm I was thinking more along the lines since this is a visual variation of the card component, it seems more appropriate to be a modifier class on the parent vs a state on the header